maintainers/maintainer-list.nix

```diff
@@ -3481,9 +3481,9 @@
     githubId = 5493775;
     name = "Ente";
   };
-  enzime = {
+  Enzime = {
     email = "enzime@users.noreply.github.com";
```

Comment:
That email should point to something else except the github one otherwise we have no other means to contact you.